exercises/practice/kindergarten-garden/.meta/example.c

```diff
@@ -0,0 +1,46 @@
+#include "kindergarten_garden.h"
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+static int student_to_index(const char *student)
+{
+   static const char *students[] = { "Alice",  "Bob",    "Charlie", "David",
+                                     "Eve",    "Fred",   "Ginny",   "Harriet",
+                                     "Ileana", "Joseph", "Kincaid", "Larry" };
+   for (size_t i = 0; i < sizeof(students); ++i) {
```

Comment:
`sizeof` gives you the size of `students` in bytes, not the number of elements.

Reply:
```suggestion
   for (size_t i = 0; i < sizeof(students)/sizeof(students[0]); ++i) {
```

Reply:
Thanks for the suggestion @ryanplusplus , that's exactly what I had in mind. Just too little practice with C, I actually _know_ what `sizeof` does ....